.github/workflows/build.yml

```diff
@@ -0,0 +1,114 @@
+---
+name: Create packages and test installation
+
+on:
+  pull_request:
+
+jobs:
+  centos7:
```

Comment:
Yes, was thinking on looking into this, will make some quick tests to see if I can get it working, otherwise it's something that could be added later.

Reply:
In fact it's not that straightforward, as for 8 and 9 the container is uay.io/centos/centos:stream8 not uay.io/centos/centos:8 (and we have el8 not elstream8 in the rpm package).

Reply:
So I kept the CentOS7 case out of the matrix, otherwise not sure how to implement this. :/